proxy.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+
+	"github.com/cloudflare/unsee/internal/alertmanager"
+	"github.com/gin-gonic/gin"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func newProxyServer(domain string) *httputil.ReverseProxy {
```

Comment:
You may wish to take an `http.RoundTripper` (to set as `ReverseProxy.Transport`) as an option here so you can observe and influence the behavior of the proxy in test.

Reply:
Not sure if that's really needed for tests, I got some now and didn't need it for that